configuration-sample/ods-core.env.sample

```diff
@@ -369,3 +369,24 @@ ADDITIONAL_JIRA_PROJECT_TEMPLATE_PROPERTIES="# ---------------------------------
 
 # All params defined in the template https://github.com/opendevstack/ods-core/blob/master/ods-provisioning-app/ocp-config/dc.yml
 # can be overwritten by specifying the param in this file.
+
+########
+# Aqua #
+########
+# Enable or disable Aqua at the cluster level
+AQUA_ENABLED=false
+
+# The URL to the Aqua service
+AQUA_URL=
+
+# The name of a container image registry in Aqua
```

Comment:
Maybe give an example here? Is this host only? or a URL with scheme?

Reply:
Added an example as we have in the odsbox after installing there Aqua

Reply:
@jorge-romero I strongly suggest you define any defaults inside the configuration. can you please change that? (simple, easy to understand, and no hidden assumptions)

Reply:
@metmajer This is the current configuration, unless there will be another one that should be set with a default value we will keep those in this way:

# Enable or disable Aqua at the cluster level
AQUA_ENABLED=false

# The URL to the Aqua service, f.e. using a local service in OCP http://<aqua-service>.aqua.svc.cluster.local:8080
# using a hosted instance of Aqua in OCP => aqua service route https://<aqua-route>.<public-cluster-url>/
AQUA_URL=

# # The name of a the internal registry defined in Aqua to keep the track of the images scanned
AQUA_REGISTRY=

# The name of a secret containing credentials (username and password) of an Aqua user that has scanner rights
AQUA_SECRET_NAME=aqua-user-with-password

# Comma separated list of email addresses to provide feedback in case there are problems using Aqua
AQUA_ALERT_EMAILS=

# The name of a Nexus repository to store the reports generated by Aqua
AQUA_NEXUS_REPOSITORY=leva-documentation

AQUA_URL depends on were Aqua is deployed and the url provided to access aqua
AQUA_REGISTRY is the internal registry configured in Aqua
AQUA_ALERT_EMAIL will depend on the recipients that will be decide to receive notifications

Reply:
@jorge-romero must have been me then. When I checked the file last time, no default values were in there, so I expected them to be hard-coded in the codebase. Thanks!